exercises/practice/affine-cipher/.meta/Sources/AffineCipher/AffineCipherExample.swift

```diff
@@ -0,0 +1,110 @@
+
```

Comment:
I notice some of the files have an extra line in the beging of the file, feels like that shouldn't be there

Reply:
No specific reason, it's just my old habbit, removed.